src/schema/objects/common_principles.yaml

```
@@ -46,6 +46,9 @@ dataset:
   description: |
     A set of neuroimaging and behavioral data acquired for a purpose of a particular study.
     A dataset consists of data acquired from one or more subjects, possibly from multiple sessions.
+derivative:
+  display_name: derivative dataset
+  description: If a dataset is derived from at least one other valid BIDS dataset, then it is a derivative dataset.
```

why to accent on "valid"? isn't it somewhat assumed since otherwise it is not a BIDS dataset per se? 
Also we are talking about BIDS derivative dataset, right?
Also I do not see need for a conditional here.

hence I would suggest

```suggestion
  description: A BIDS dataset derived from at least one other BIDS dataset.
```